app/services/payment_provider_customers/stripe_service.rb

```diff
@@ -103,12 +103,12 @@ def generate_checkout_url
       )
 
       result.checkout_url = res['url']
-
+      
       SendWebhookJob.perform_later(
         'customer.checkout_url_generated',
         customer,
         checkout_url: result.checkout_url,
-      )
+      ) if send_webhook
```

Comment:
Style/MultilineIfModifier: Favor a normal if-statement over a modifier clause in a multiline statement. (https://rubystyle.guide#no-multiline-if-modifiers)